pkg/agent/cniserver/server.go

```diff
@@ -453,12 +453,12 @@ func (s *CNIServer) CmdAdd(ctx context.Context, request *cnipb.CniCmdRequest) (*
 		// Rollback to delete configurations once ADD is failure.
```

Comment:
Not your code, but change "is failure" to "fails"?

Reply:
Done